### ISSUE TITLE
[All] Command line option for verbose language loading

### ISSUE THREAD

**Issue by RAnders00** [MinecraftForge/MinecraftForge#1833]
It would be great if Forge could add a command line option to output info about translation.
I would imagine:
###### Level 3 Verbose logging (logs success as well)
`Successfully translated "botania.entry.apothecary" to "Blütenblatttiegel" (lang: "de_DE")`
`Language fallback: "botania.entry.apothecary" not found in "de_DE", falling back to "en_US"`
`Unable to translate "botania.entry.apothecary"! Not found in neither "de_DE" nor "en_US"!`
###### Level 2 Verbose logging (logs fallbacks)
`Language fallback: "botania.entry.apothecary" not found in "de_DE", falling back to "en_US"`
`Unable to translate "botania.entry.apothecary"! Not found in neither "de_DE" nor "en_US"!`
###### Level 1 Verbose logging (logs nowhere-found-errors)
`Unable to translate "botania.entry.apothecary"! Not found in neither "de_DE" nor "en_US"!`
###### Level 3 Verbose logging (logs success as well)
`Successfully translated "botania.entry.apothecary" to "Blütenblatttiegel" (lang: "en_US")`
`Unable to translate "botania.entry.apothecary"! Not found in "en_US"!`
###### Level 1 and 2 Verbose logging (logs nowhere-found-errors)
`Unable to translate "botania.entry.apothecary"! Not found in "en_US"!`
## Why I am requesting this
This would make localization of mods that much easier. You could easily find out what has not been translated yet and it would be much quicker and easier to create localization files.

**Comment by LexManos** [comment]
This type of logging would just cause spam. It's simpler either to just write a mod that loops through the en_US {which all mods should provide a full entry set for} and try and translate them
Relying them to show up naturally in the game isn't really conducive to debugging as well you're not guaranteed to get all the entries.

Maybe some other approach that makes the game dump this data to a separate 'translation needed' log or something. But just spitting things out to the main log is just spammy and hard to follow.
Beyond this, the translations are 'flattened' at runtime so there is no way to know if something isn't in one language or another, it's simply 'Was this translated or not!'

A different approach is needed.
